srcalc/config/checkstyle/checkstyle.xml

```diff
@@ -40,6 +40,21 @@
     <module name="JavadocStyle"/>
     <module name="NeedBraces"/>
     <module name="OuterTypeNumber"/>
+    <module name="LineLength">
+      <property name="ignorePattern" value="http://"/>
+      <property name="max" value="110"/>
+      <property name="tabWidth" value="4"/>
+    </module>
+    <module name="JavadocMethod">
+      <property name="scope" value="package"/>
+      <property name="allowUndeclaredRTE" value="true"/>
+      <property name="allowMissingParamTags" value="true"/>
+      <property name="allowMissingReturnTag" value="true"/>
+      <property name="suppressLoadErrors" value="true"/>
+    </module>
   </module>
   <module name="FileTabCharacter"/>
```

Comment:
Since we have had some inconsistencies in the past, do we want this to report on each line that has tab characters rather than just the files?